sk0-front/src/app/app.component.scss

```diff
@@ -0,0 +1,3 @@
+.capitalize-text {
```

Comment:
do we need shouting text?

Reply:
Only the fist letter was uppercase, but  now found a better way to get around ionic button's default uppercase lettering.

Reply:
Let it style the ionic way is my opinion